packages/opentelemetry-instrumentation-anthropic/tests/test_messages.py

```
@@ -2807,3 +2807,36 @@ async def test_async_anthropic_beta_message_stream_manager_legacy(
+    for _ in response:
+        pass
+
+    spans = span_exporter.get_finished_spans()
+    anthropic_span = spans[0]
+
+    assert anthropic_span.attributes[GenAIAttributes.GEN_AI_USAGE_INPUT_TOKENS] == 343
+    assert (
+        anthropic_span.attributes[GenAIAttributes.GEN_AI_USAGE_OUTPUT_TOKENS]
+        + anthropic_span.attributes[GenAIAttributes.GEN_AI_USAGE_INPUT_TOKENS]
+        == anthropic_span.attributes[SpanAttributes.GEN_AI_USAGE_TOTAL_TOKENS]
+    )
```

Ruff (0.15.9)

[warning] No newline at end of file

Add trailing newline

(W292)